internal/store/custom_dns.go

```diff
@@ -56,7 +57,12 @@ func (s *Store) DisableCustomDNS(projectID string) (*atlasv2.AWSCustomDNSEnabled
 		customDNSSetting := &atlasv2.AWSCustomDNSEnabled{
 			Enabled: false,
 		}
-		result, _, err := s.clientv2.AWSClustersDNSApi.ToggleAWSCustomDNS(s.ctx, projectID).AWSCustomDNSEnabled(*customDNSSetting).Execute()
+		result, _, err := s.clientv2.AWSClustersDNSApi.ToggleAWSCustomDNS(s.ctx, projectID).
+			AWSCustomDNSEnabled(*customDNSSetting).Execute()
+		apiError := sdkv2.APIError(err)
+		if apiError != nil {
+			return nil, fmt.Errorf("%v", apiError)
```

Comment:
```suggestion
		if apiError := sdkv2.APIError(err); apiError != nil {
			return nil, fmt.Errorf("%v", apiError)
```

Reply:
is there any reason why you are using `fmt.Errorf` without adding any additional context to the error instead of returning directly the error?

Reply:
Could you share the logic of `sdkv2.APIError` in the go SDK? Thanks!